README.md

```diff
@@ -67,3 +67,23 @@ sudo systemctl enable --now libvirtd
 sudo usermod --append --groups libvirt `whoami`
 vagrant up --no-parallel
 ```
+
+## Developer Workstation Setup (Debian)
```

Comment:
this is great to have, thank you. a brief (just two to four sentences) describing the testing strategy and any other preconditions that are hard to express in a few bash lines might be helpful.

For instance, something like "We use Ansible Molecule for testing. This role includes the necessary vagrant and molecule configuration that accesses XXX other resources on XYZ machines. It assumes you have access to XYZ machines."

I'm not really familiar with molecule or vagrant, so I'm not sure if that's redundant, but given all the ip addresses that show up in the vagrant file I'm guessing you need some kind of credentials to actually be successful in running the tests.

Reply:
I did not write this stuff originally and am still figuring out the strategy.  @bkus would be best to add that commentary.  This work is a precondition for doing similar testing for RouterOS.  I am not comfortable writing up the strategy nor preconditions yet.

Reply:
I have added a brief comment that will need more detail when I have written the routeros_common testing.  It would probably also benefit from some commentary from Bart.